server/src/main/java/org/apache/druid/metadata/SQLMetadataStorageActionHandler.java

```diff
@@ -439,6 +449,35 @@ public Void withHandle(Handle handle)
     );
   }
 
+  @Override
+  public void removeTasksOlderThan(final long timestamp)
+  {
+    connector.retryWithHandle(
+        (HandleCallback<Void>) handle -> {
+          DateTime createdTime = DateTimes.utc(timestamp);
```

Comment:
I do not like this variable name, it's a time point not a created time. Maybe `DateTime dateTime = DateTimes.utc(timestamp);`
 